src/types.rs

```diff
@@ -214,6 +219,12 @@ pub struct RawObjectInfo {
     pub debug_file: Option<String>,
 
     /// Absolute address at which the image was mounted into virtual memory.
+    ///
+    /// We do allow the image addr to be skipped if it's zero.  This is
+    /// because for instance systems like WASM do not actually require an
+    /// image to be mounted at a specific address.  This makes sense mostly
+    /// when using `in_module` lookups.
+    #[serde(default)]
```

Comment:
So wasm requests would omit this and serde would fill this in wit `0`?  Would it be clearer for the code to change this to `Option`?  Or even go further and introduce a `RawWasmFrame`?

Reply:
I prefer it to just be optional on the protocol. Makes the code cleaner.